screens/auth/home/tabs/settings/SettingsScreen.tsx

```diff
@@ -1,18 +1,35 @@
-import {useColorScheme, View, StyleSheet} from 'react-native';
+import {useColorScheme, View, StyleSheet, Pressable} from 'react-native';
 import {getAppColorStyles} from '@styles/colors';
+import {GestureHandlerRootView} from 'react-native-gesture-handler';
+import {useRef} from 'react';
+import type BottomSheet from '@gorhom/bottom-sheet';
 
 import {ToggleThemeTile} from './components/ToggleThemeTile';
 import {LogoutTile} from './components/LogoutTile';
+import {DeleteAccountTile} from './components/DeleteAccountTile';
+import {DeleteAccountBottomSheet} from './components/DeleteAccountBottomSheet';
 
 export default function SettingsScreen() {
   const colorScheme = useColorScheme();
   const {backgroundStyle} = getAppColorStyles(colorScheme);
+  const bottomSheetRef = useRef<BottomSheet>(null);
 
   return (
-    <View style={[styles.containerLayout, backgroundStyle]}>
-      <ToggleThemeTile />
-      <LogoutTile />
-    </View>
+    <Pressable
+      style={{flex: 1}}
+      onPress={() => bottomSheetRef.current?.close()}>
+      <GestureHandlerRootView style={[styles.containerLayout, backgroundStyle]}>
```

Comment:
This component [should go to the `App`](https://docs.swmansion.com/react-native-gesture-handler/docs/guides/migrating-off-rnghenabledroot/#update-your-js-code) (or similar) that would wrapp the whole app